src/hooks/useViewportRows.ts

```diff
@@ -176,15 +174,9 @@ export function useViewportRows<R>({
 
   const overscanThreshold = 4;
```

Comment:
Should we consider adding `overscanRowCount` and `overscanColumnCount` to customize this behavior similar to [react-window](https://react-window.vercel.app/#/api/FixedSizeGrid)?

Reply:
I'd rather not, it's more APIs we'd have to support, we should reserve the right to change them, like how `RENDER_BATCH_SIZE` is being removed in this PR.